src/wta/main.py

```diff
@@ -50,10 +50,12 @@ def run(log_path: Path,
     transitions_data = activity_transitions.identify(
         log, parallel_activities, parallel_run, log_ids=log_ids, calendar=calendar, group_results=group_results)
 
-    if group_results:
-        return TransitionsReport(transitions_data, log, log_ids)
-    else:
-        return transitions_data
+    # if group_results:
+    #     return TransitionsReport(transitions_data, log, log_ids)
+    # else:
+    #     return transitions_data
+
+    return transitions_data
```

Comment:
Please, remove the commented part if it's not needed anymore. We try to keep code as clean as we can.